src/SFA.DAS.EAS.Web/Views/Home/Privacy.cshtml

```diff
@@ -323,13 +323,13 @@
             Most services we link to are run by different government departments, eg HM Revenue and Customs (<abbr title="Her Majesty’s Revenue and Customs">HMRC</abbr>) or the Home Office. These services may set additional cookies and, if so, will have their own cookies policy.
         </p>
 
-        <h3 class="heading-medium">
+        @*<h3 class="heading-medium">
```

Comment:
Do we need to replace this with a link to smart survey rather than commenting the section out

Reply:
Done. Replaced link to the new survey and updated text to match